src/content/translations/ja/build/index.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Once again, you're not following instructions. This file should be deleted.

Please follow the instructions in the issue: https://github.com/ethereum/ethereum-org-website/issues/1484